### PR TITLE
Document missing default hotkeys

### LINKED ISSUE #181: Please document all hotkeys on the main github information page
For the love of God, please do it. I know there are more hotkeys than there are listed. Is there no settings window that allows you to remap buttons? One shortcut I found by sheer trying is F9 to /exit. Or at least I believe it's from APT. Is it normal that right-clicking the Icon in the task bar does not open any context menu? No setting to exit or open settings? 

The most infuriating thing is that for some unknown reason some overlay opened in the top left of my screen with the options to add a countdown to the center of my screen. I brought it up by sheer mistake (just opening chat with Enter and started typing...) and now I don't know how to remove it because the underlying issue is: the full documentation of all shortcuts is missing. Please edit the information page, would be very helpful. 

PS: Great fan of your application. It makes price checking fast and much more fun than before. You've built a great program!

### PR DESCRIPTION
**Pull request by brian-brazil**
Fixes #181

I only see two hotkeys that are missing from the defaults. I also mentioned how to get to the settings, and what the overlay can do.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,9 +61,14 @@
 - Press `Ctrl + Alt + D` to check the price of the item\
   *Opens a window, but does not hide it automatically when moving the mouse*
 - Press `F5` to go to hideout
+- Press `F9` to exit to character selection
 - `Ctrl + MouseWheel` to scroll through stash tabs\
   *You can also disable "Mousewheel Zoom" in PoE settings (Options -> UI -> 3-rd check mark starting from the bottom)*
 - `Alt + W` to open the item on Wiki
+- `Crtl + Shift + Space` to open overlay
```
Comment (five-iron) [inline README.md:68]: This seems to be just Shift + Space (no Ctrl).

Comment (brian-brazil) [reply README.md:68]: That sounds either like a bug, or the default changed. What does it say in the Hotkeys settings?

Comment (five-iron) [reply README.md:68]: Looks like it's been that way a while: https://github.com/SnosMe/awakened-poe-trade/blob/master/src/ipc/types.ts#L66

Comment (brian-brazil) [reply README.md]: So it is, I've fixed the PR.

I think I'd changed my setting as I was hitting it accidentally,

**Comment by SnosMe** [comment]
https://snosme.github.io/awakened-poe-trade/quick-start

**Pull request closed without merging**